sources/main.cpp

```diff
@@ -10,7 +10,9 @@ int	main(int ac, char **av)
 	(void) av;
 
 	ServerManager::ServerBlocks	blocks;
-	blocks.push_back((ServerConfig){Network::sToIPV4Packed("127.0.0.1"), 5050});
+	ServerConfig				config;
+
+	blocks.push_back(config);
```

Comment:
You have to read the config file to find out how many blocks there are. Also since this is parsing you need to get argv[1} and use it as path for parsing. How are you doing parsing without knowing the file you have to parse?Also in this line This may have multiple blocks. inside of it not only one depending on how many server blocks are in the conf file

Reply:
No parsing is being done atm. The class constructors to load the configs from a file are currently being implemented, and the actual parsing will be saved for last. For now, this is the only way to instantiate objects from the ServerConfig class (ie with the default constructor and hardcoded settings)